packages/nextjs/src/config/webpack.ts

```diff
@@ -435,6 +436,33 @@ async function addSentryToClientEntryProperty(
   return newEntryProperty;
 }
 
+/**
+ * Make sure the instrumentation file has a `onRequestError` Handler
+ *
+ * @param projectDir The root directory of the project, where config files would be located
+ */
+function warnAboutMissingonRequestErrorHandler(projectDir: string): void {
```

Comment:
It would be nice if we could actually only log the warning when there even is an instrumentation file and if there is not, warn about not having it.